content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,317 @@
+ 403         | The client is not allowed to perform the operation                                                    | error    | forbidden
+ 404         | The Group requested is not found                                                                      | error    | not-found
+ 422         | `_typeFilter` is provded with an invalid or unsupported value                                         | error    | invalid
+ 429         | There is already an export job in-progress for the client and tenant combination                      | error    | throttled
+ 500         | An internal error occurred.                                      									 | fatal    | exception
+
+
+## Status Request
+
+Checks the status of the export job.
+Use the URL returned in the `Content-Location` header from the kick off request to retrieve the status of a bulk data export:
+
+    GET  <URL from Content-Location header>
+
+_Notes_
+
+* Clients should follow an Exponential Backoff approach when polling for status as outlined in the [bulk data export specification.](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request)
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* Scopes are not checked in this request. The authorization token must be valid, and the client in the token must match the client that requested the export.
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+### Example
+
+#### In-Progress Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 202 Accepted
+    X-Progress: in-progress
+    Retry-After: 600
+
+#### Complete Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 200 OK
+    Expires: Mon, 22 Jul 2025 23:59:59 GMT
+    Content-Type: application/json
+
+<%= json(:R4_BULK_EXPORT_COMPLETE) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The following errors and OperationOutcomes may be returned in response to the status request:
+
+ HTTP Status | Cause                                                      | Severity | Code
+-------------|------------------------------------------------------------|----------|----------
+ 404         | The export job cannot be found                             | error    | not-found
+ 406         | The value supplied to the `Accept` header is not supported | error    | invalid
+ 500         | An internal error occurred.                                | fatal    | exception
+
+## File Retrieval Request
+
+Once an export is complete, the status request will return file URLs which may be used to download the exported resources.
+
+    GET <File URL from status response
```

Comment:
missed `>`

Reply:
Thank you, I have added it here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/780d6472fc27b87e37ef7361457f44e8da969b69#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR184